python/lsst/analysis/tools/actions/plot/histPlot.py

```diff
@@ -163,7 +217,7 @@ def _makeAxes(self, fig):
             ncols = 2
         nrows = int(np.ceil(num_panels / ncols))
 
-        gs = GridSpec(nrows, ncols, left=0.13, right=0.99, bottom=0.1, top=0.88, wspace=0.25, hspace=0.45)
+        gs = GridSpec(nrows, ncols, left=0.12, right=0.99, bottom=0.1, top=0.88, wspace=0.31, hspace=0.45)
```

Comment:
I'd probably also recommend reformatting this line too:
```
        gs = GridSpec(
            nrows,
            ncols,
            left=0.12,
            right=0.99,
            bottom=0.1,
            top=0.88,
            wspace=0.31,
            hspace=0.45,
        )
```

Reply:
Apologies, this one slipped through my net. I had my local `black` formatter set up incorrectly with regards line length. You can change this back to:
```
gs = GridSpec(nrows, ncols, left=0.12, right=0.99, bottom=0.1, top=0.88, wspace=0.31, hspace=0.45)
```
if you'd prefer? Sorry for the mix-up - I thought I'd caught all of these comments and removed them, but missed this one.

Reply:
Oh, I figured you were giving me your preferred aesthetic for formatting (the PR checks had all passed, so we already knew black was “happy”).  I don’t really have a preference on this one…it’s so close to being at max length that having it this way is arguably preferable in case anyone adds to it, so I’m inclined to leave the update?

Reply:
I had to get back in there, so I did switch it back...